haystack/preview/components/audio/whisper_local.py

```diff
@@ -4,9 +4,12 @@
 from pathlib import Path
 
 import torch
-import whisper
 
 from haystack.preview import component, Document, default_to_dict, default_from_dict, ComponentError
+from haystack.preview.lazy_imports import LazyImport
+
+with LazyImport("Run 'pip install openai-whisper'") as whisper_import:
+    import whisper
```

Comment:
Very good 💯  but ideally these should've gone in a different PR

Reply:
Argh you're right...